### PR TITLE
Create 2024-04-17.md

### PR DESCRIPTION
**Pull request by sthagen**
Meeting minutes of 2024-04-17.

### CODE REVIEW

File: meeting-minutes/2024-04-17.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    August 21, 2024
+    September 18, 2024
+    October 16, 2024 (daylight saving times still active)
+    November 29, 2024 (normal time switch back done, same time on wall clock)
+    December 18, 2024
+    ```
+
+# 3. Discussion
+
+* Workspace at OASIS still not completely usable, but al OpenEOX TC meetings of 2024 are already scheduled and available.
```
Comment (tschmidtb51) [inline meeting-minutes/2024-04-17.md:97]:
```suggestion
* Workspace at OASIS still not completely usable, but all OpenEOX TC meetings of 2024 are already scheduled and available.

File: meeting-minutes/2024-04-17.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+* CSAF Workshop May 14-16 (CSAF Workshops (oasis-open.github.io))
```
Comment (tschmidtb51) [inline meeting-minutes/2024-04-17.md:98]:
```suggestion
* CSAF Workshops May 14-16 in Germany (for details see [workshop.csaf.io](https://workshop.csaf.io))

File: meeting-minutes/2024-04-17.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+* Issues
+  * "[Define Common Taxonomy #11](https://github.com/oasis-tcs/openeox/issues/11)"
+  * "[One Use Case w/ portable schema #14](https://github.com/oasis-tcs/openeox/issues/14)"
+  * "[lifecycle schema proposal #15](https://github.com/oasis-tcs/openeox/issues/15)" 
+    * All discuss how to focus on definitive set of labels
+      * without trying to please all current possible labels
+      * but considering proven additional dimensions (like three levels of support instead of a single level)
+      * focus on unambiguous semantics of the labels and not so much on the text of labels
+      * distinguish the terms description and definition
```
Comment (tschmidtb51) [inline meeting-minutes/2024-04-17.md:110]:
```suggestion
      * distinguish the terms `description` and `definition`

**Review by tschmidtb51 — COMMENTED** [review]
@sthagen Thank you for the meeting minutes - very well done!

Just some nits from my side.

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Review by justmurphy — APPROVED** [review]
Looks good!

**Review by santosomar — APPROVED** [review]

**Comment by santosomar** [comment]
[Motion to approve OpenEoX April 2024 Meeting Minutes](https://groups.oasis-open.org/discussion/motion-to-approve-openeox-april-2024-meeting-minutes)

**Pull request merged**